README.md

```diff
@@ -605,13 +606,15 @@ Then, you will need to use the `e` option to manually choose which lines to add.
 <a href="unstaging-edits-and-staging-the-unstaged"></a>
 ### I want to stage my unstaged edits, and unstage my staged edits
 
-This is tricky. The best I figure is that you should stash your unstaged edits. Then, reset. After that, pop your stashed edits back, and add them.
+In many cases, You should unstage all files and then pick file you want and commit it.
+But assume that's what you want,You can create a temp commit to store your stage files, stage your unstaged files and stash them, Then, reset the temp commit and stash pop.
 
 ```sh
-$ git stash -k
```

Comment:
We should add a note that git stash or git stash k doesn't work with newfiles, only with edits.

Reply:
you may need `-u` option

Reply:
Can you add something for that?

Reply:
I think It's already [here](https://github.com/k88hudson/git-flight-rules#stash-all-edits) :p